sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md

```diff
@@ -1019,6 +1020,24 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -SharePointAddInsDisabled
+
+Allows or disallows disable the SharePoint add-ins app.
+
+When the value is set to false, the SharePoint add-ins app can still be used. If the value is set to true, sharepoint will no longer support any features related to add-ins.
```

Comment:
reword this similar to how `BccExternalSharingsInvitationList` has done it on this page